codespell_lib/data/dictionary.txt

```diff
@@ -8110,10 +8124,12 @@ otherwsie->otherwise
 othewice->otherwise
 othewise->otherwise
 othewize->otherwise
+otho->otoh
```

Comment:
say what?

Reply:
acronym, OTOH (on the other hand)

Reply:
oh right. heh. I guess we should keep it